docs/index.md

```diff
@@ -0,0 +1,187 @@
+# Regula
+
+## Overview
+
+The "Regula" project addresses the problems caused by the accumulation of simple logic that changes frequently in line with business requirements.. This introduces strains on the development cycle by reducing the velocity. In concrete terms, developers can find themselves always chasing the business rather than focusing on core logic which defines the business.
```

Comment:
You have an extra full stop (`.`) at the end of the first sentence.

Reply:
Rah geez, sorry for bothering you to post a comment for this.